en/concept/core_components.md

```diff
@@ -0,0 +1,91 @@
+| Battery (Library)                                                                  |                                      |                                                  |
+| Flight Task (Library)                                                              |                                      |                                                  |
+| Matrix (Library)                                                                   |                                      |                                                  |
+| Autotuning (Library)                                                               |                                      |                                                  |
+| [Github Actions (Continuous Integration)][Github Actions (Continuous Integration)] |                                      |                                                  |
+| [Jenkins (Continuous Integration)][Jenkins (Continuous Integration)]               |                                      |                                                  |
+| Linux (Toolchain)                                                                  |                                      |                                                  |
+| Mac OS (Toolchain)                                                                 |                                      |                                                  |
+| [Windows (Toolchain)][Windows (Toolchain)]                                         | WSL2                                 |                                                  |
+| [ROS2][ROS2]                                                                       |                                      |                                                  |
+| [Gazebo Classic (Simulation)][Gazebo Classic (Simulation)]                         |                                      |                                                  |
+| [Gazebo (Simulation)][Gazebo (Simulation)]                                         |                                      |                                                  |
+| Flightgear (Simulation)                                                            |                                      |                                                  |
+| JSBSim (Simulation)                                                                |                                      |                                                  |
+| jMAVSim (Simulation)                                                               |                                      |                                                  |
+| SIH (Simulation)                                                                   |                                      |                                                  |
```

Comment:
The are all community maintained. I think it might be worth listing these separately. Only SIH has a maintainer, of sorts.

```suggestion
```